packages/http/src/validator/validators/body.ts

```diff
@@ -11,6 +11,8 @@ import { JSONSchema } from '../../types';
 import { body } from '../deserializers';
 import { validateAgainstSchema } from './utils';
 import { ValidationContext, validateFn } from './types';
+// @ts-ignore no typings
+import * as mergeAllOf from 'json-schema-merge-allof';
```

Comment:
`@ts-ignore no typings` feels wrong.  Is there a better way?  I couldn't find any types for the Stoplight branch of `json-schema-merge-allof` or the current version of `json-schema-merge-alllof`.